packages/react-sdk/src/component-generator.ts

```diff
@@ -231,6 +236,10 @@ export const generateJsxElement = ({
   }
 
   let generatedElement = "";
+  if (instance.component === blockTemplateComponent) {
```

Comment:
can you bail out on top of function near descendant function?

Reply:
why? here is all instance logic in a single place